agenthub/codeact_agent/system_prompt.j2

```diff
@@ -19,9 +19,9 @@ the assistant should retry running the command in the background.
 
 [3] The assistant can edit files using <file_edit> by setting the file path and providing a draft of the new file content. The draft file content does not need to be exactly the same as the existing file content; the assistant may skip some lines and only include the parts that need to be changed.
 
-When editing large file (e.g., > 300 lines), the assistant should specify the range of lines to be edited by setting `start` and `end` (1-indexed, both inclusive). For example, `<file_edit path="/path/to/file.txt" start=1 end=-1>` means the assistant will edit the whole file (from line 1 to the end of the file). `start=1` and `end=-1` are the default values, so the assistant can omit them if they are the same as the default values.
+IMPORTANT: When editing large file (e.g., > 300 lines), the assistant SHOULD SPECIFY the range of lines to be edited by setting `start` and `end` (1-indexed, both inclusive). For example, `<file_edit path="/path/to/file.txt" start=1 end=-1>` means the assistant will edit the whole file (from line 1 to the end of the file). `start=1` and `end=-1` are the default values, so the assistant can omit them if they are the same as the default values.
```

Comment:
We could even go with MUST SPECIFY, why not 😄

Reply:
I REPEAT! YOU MUST ... 😄